src/unix/linux_like/linux/arch/generic/mod.rs

```diff
@@ -277,6 +277,7 @@ cfg_if! {
         pub const RLIMIT_NICE: ::c_int = 13;
         pub const RLIMIT_RTPRIO: ::c_int = 14;
         pub const RLIMIT_RTTIME: ::c_int = 15;
+#[deprecated(since = "0.2.64", note = "Not stable across OS versions")]
```

Comment:
Looks like some kind of whitespace problem here.

Reply:
Oh?  But I ran `rustfmt` and the `ci/style.rs` says `good style!`.

Can you offer more details about the whitespace problem?

Reply:
The `#[deprecated...]` should have enough leading space to line up with the line it deprecates.  That's how the rest of the crate does it.

Reply:
Okay -- got it, thanks.